views/partials/project/links.blade.php

```diff
@@ -0,0 +1,14 @@
+<div class="o-grid-12">
+    @card
+        <div class="c-card__body u-pt-1">
+            <h4>{{$lang['linksLabel']}}</h4>
+            <ul>
+                @foreach ($project['links'] as $link)
+                    <li>
+                        <a target="_blank" href="{{ $link['url'] }}">{{ $link['title'] }}</a>
```

Comment:
Add rel="noopener" to links opened as _blank. See https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener and https://sonarcloud.io/project/security_hotspots?id=helsingborg-stad_api-project-manager-integration&pullRequest=43&hotspots=AYUpvZ6Sh0kgxdFwZbka